stdlib/public/SDK/Foundation/Data.swift

```diff
@@ -939,38 +907,47 @@ public struct Data : ReferenceConvertible, Equatable, Hashable, RandomAccessColl
     /// - parameter count: The number of bytes to copy.
     public init(bytes: UnsafeRawPointer, count: Int) {
         _backing = _DataStorage(bytes: bytes, length: count)
+        _sliceRange = 0..<count
```

Comment:
This initialization gets repeated a lot. What about moving it to the declaration of _sliceRange, and just setting it differently in the few init()s where it's not 0..<count?